lib/kino_db/connection_cell.ex

```diff
@@ -11,20 +11,32 @@ defmodule KinoDB.ConnectionCell do
 
   @impl true
   def init(attrs, ctx) do
-    default_type = default_db_type()
-    default_port = @default_port_by_type[default_type]
+    type = attrs["type"] || default_db_type()
 
-    fields = %{
-      "variable" => Kino.SmartCell.prefixed_var_name("conn", attrs["variable"]),
-      "type" => attrs["type"] || default_type,
+    fields =
+      Map.merge(fields(type, attrs), %{
```

Comment:
When the database type change we will set the other fields too, so we may as well initialize all of them for simplicity?

Reply:
much more simple